cmake/scripts/copydlls.cmake

```diff
@@ -8,10 +8,10 @@
 
 # this script is created cause we do not know configuration in multiconfiguration generators at cmake configure phase ;)
 
-if ("${CONF}" STREQUAL "Release")
-    set(DLL ${DLL_RELEASE})
-else () # Debug
+if ("${CONF}" STREQUAL "Debug")
     set(DLL ${DLL_DEBUG})
+else ()
+    set(DLL ${DLL_RELEASE})
```

Comment:
Yes. "debug" dependencies should be used only for Debug builds. "release" for all others.